CONTRIBUTING.md

```diff
@@ -32,28 +32,34 @@ have prepared these guidelines for you:
 
 ## How Do I Contribute?
 
-There are different ways to contribute, each comes with a different levels
-of tasks, such as:
+There are many ways to contribute:
 
-- Report a bug.
-- Request a feature you think would be great for verdaccio.
-- Fix bugs.
-- Test and triage reported bugs by others.
-- Work on requested/approved features.
-- Improve the codebase (lint, naming, comments, test descriptions, etc...)
+- Report a bug
+- Request a feature you think would be great for Verdaccio
+- Fix bugs
+- Test and triage bugs reported by others
+- Work on requested/approved features
+- Improve the codebase (linting, naming, comments, test descriptions, etc...)
 
-Verdaccio has several areas of involvement, which might fit you better, eg:
+The Verdaccio project is split into several areas:
 
-- **Core**: The [core](https://github.com/verdaccio/verdaccio) is the main repository, built with **Node.js**.
-- **Website**: we uses **Gatsby.js** for the **website** and if you are familiar with this technology, you might become the official webmaster.
-- **User Interface**: The [user Interface](https://github.com/verdaccio/ui) is based in **react** and **material-ui** and looking for front-end contributors.
-- **Kubernetes and Helm**: Ts the official repository for the [**Helm chart**](https://github.com/verdaccio/charts).
+- **Core**: [core](https://github.com/verdaccio/verdaccio) is the main
+repository, built with **Node.js**
+- **Website**: we use **Gatsby.js** for the **website** and if you are familiar
```

Comment:
Please,  replace `Gatsby.js` by `Docusaurus`, we decided not to continue with `Gatsby.js` for some reasons #1862